pytorch_lightning/core/lightning.py

```diff
@@ -463,8 +463,11 @@ def training_step(self, *args, **kwargs):
             Any of.
 
             - :class:`~torch.Tensor` - The loss tensor
-            - `dict` - A dictionary. Can include any keys, but must include the key 'loss'
-            - `None` - Training will skip to the next batch
+            - ``dict`` - A dictionary. Can include any keys, but must include the key ``loss`
+            - ``None`` - Training will skip to the next batch
+
+        Note:
+            Returning ``None`` is currently not supported for multi-GPU or TPU, or with half precision enabled.
```

Comment:
```suggestion
            Returning ``None`` is currently not supported for multi-GPU or TPU, or with 16-bit precision enabled.
```
with both amp and native??

Reply:
i assume you mean apex and native, yes